prepare/metrics/llm_as_judge/rating/llama_3_cross_provider_table2text_template.py

```diff
@@ -0,0 +1,32 @@
+from unitxt import add_to_catalog
+from unitxt.inference import CrossProviderInferenceEngine
+from unitxt.llm_as_judge import LLMAsJudge
+from unitxt.random_utils import get_seed
+
+model_list = ["llama-3-70b-instruct"]
+format = "formats.llama3_instruct"
```

Comment:
I think you should use "formats.chat_api" like in

https://www.unitxt.ai/en/latest/catalog/catalog.metrics.llm_as_judge.rating.llama_3_70b_instruct.generic_single_turn.html

Reply:
Otherwise the formatted prompt will be nested in the user message.  [ We are going to generate an error on this in the future]